lofo/lofo_importance.py

```diff
@@ -50,7 +53,7 @@ def _get_cv_score(self, feature_to_remove):
 
         with warnings.catch_warnings():
             warnings.simplefilter("ignore")
-            cv_results = cross_validate(self.model, X, y, cv=self.cv, scoring=self.scoring, fit_params=fit_params)
+            cv_results = cross_validate(self.model, X, y, cv=self.cv, scoring=self.scoring, fit_params=fit_params, verbose=self.verbose)
```

Comment:
./lofo/lofo_importance.py:56:121: E501 line too long (136 > 120 characters)

Reply:
Also I dont think it will look good together with a progress bar and prints in between. Maybe we shouldnt have verbose here. Did it come from a need?